tests/framework/ROM/SKLearn/occ.xml

```diff
@@ -45,7 +45,9 @@
         <SKLtype>svm|SVC</SKLtype>
         <fit_intercept>True</fit_intercept>
         <normalize>True</normalize>
+        <random_state>0</random_state>
       </estimator>
+      <random_state>0</random_state>
     </ROM>
```

Comment:
Do we need to set it in both places? 
Shouldn't be enough to set it in the multiclass and send it to all the estimators?

Reply:
I think I agree with you, one is good to all the estimators.

Reply:
I do test it, removing the inside random_state does not break the test. I will remove it.